releasenotes/notes/fix-anthropic-input-tokens-unified-calculation-a1b2c3d4e5f6g7h8.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    LLM Observability: This fix resolves an issue where anthropic prompt caching caused input token counts to be the 
```

Comment:
Was prompt caching itself the cause of the issue, or was the cause in ddtrace itself? Consider rewording this to make it more about something we as library maintainers have control over.